bin/addons-linter

```diff
@@ -4,7 +4,13 @@ var path = require('path');
 
 var absoluteAppRoot = path.resolve(path.join(__dirname, '..'));
 global.appRoot = path.relative(process.cwd(), absoluteAppRoot);
-
-require('../dist/addons-linter')
-  .createInstance({runAsBinary: true})
-  .run();
+try{
+    require('../dist/addons-linter')
```

Comment:
It's odd the linter isn't catching this but these indents are off compared to our usual style. would you mind changing the layout to:

```js
try {
  require('../dist/addons-linter')
  .createInstance({ runAsBinary: true })
  .run();
} catch (err) {
  if (e.code == 'MODULE_NOT_FOUND') {
    console.log('You did not build addons-linter yet. Please run \'npm install\' and \'yarn build\' or see README.md for more information');
  }
}
```